### PR TITLE
Lint all the code!

### PR DESCRIPTION
**Pull request by flurin**
Also includes a fix where FixtureHelper was put into test_helper.rb.

### CODE REVIEW

File: test/middleware_test.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class TestMiddleware < ::Test::Unit::TestCase
     include FixtureHelper
 
     def setup
-      @app =  Proc.new { [200, {}, ["YAM"]] } # Yet another middleware
+      @app =  proc { [200, {}, ["YAM"]] } # Yet another middleware
```
Comment (flurin) [inline test/middleware_test.rb:10]: NO

Comment (hkrutzer) [inline test/middleware_test.rb:10]: i chzbgr

Comment (flurin) [inline test/middleware_test.rb:10]: NO!

**Comment by edwinvdgraaf** [comment]
One tiny thingie, would it be usefull to add a rake task for rubocop like: https://github.com/DigitPaint/roger_scsslint/blob/master/Rakefile

**Pull request merged**